.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,39 @@
+# https://github.com/golangci/golangci-lint-action#how-to-use
+name: golangci-lint
+on:
+  push:
+    branches: [main]
+  pull_request:
+permissions:
+  contents: read
+  pull-requests: read
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      # this is needed by containers/storage which we're using in gangplank
+      - name: Install deps
+        run: sudo apt-get install -y libgpgme-dev libbtrfs-dev libdevmapper-dev
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
+      - uses: actions/checkout@v3
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v3
+        with:
+          version: latest
```

Comment:
We generally pin the linter, and should do the same here to prevent uncontrolled CI regressions.

Reply:
Fixed, thanks! I initially did pin it (it's documented that way as well), but wasn't sure if the initial errors I was hitting were due to some bug, so had bumped it to `latest` to see if it went away.